enclave-runtime/src/tests.rs

```diff
@@ -503,34 +501,6 @@ fn test_non_root_shielding_call_is_not_executed() {
 	assert_eq!(funds_new, funds_old);
 }
 
-fn ensure_empty_extrinsic_vec_triggers_zero_filled_merkle_root() {
-	// given
-	let block_hash = H256::from([1; 32]);
-	let extrinsics = Vec::new();
-	let expected_call =
-		([TEEREX_MODULE, PROCESSED_PARENTCHAIN_BLOCK], block_hash, H256::default()).encode();
-
-	// when
-	let call = crate::create_processed_parentchain_block_call(block_hash, extrinsics);
-
-	// then
-	assert_eq!(call.0, expected_call);
-}
-
-fn ensure_non_empty_extrinsic_vec_triggers_non_zero_merkle_root() {
-	// given
-	let block_hash = H256::from([1; 32]);
-	let extrinsics = vec![H256::from([4; 32]), H256::from([9; 32])];
-	let zero_root_call =
-		([TEEREX_MODULE, PROCESSED_PARENTCHAIN_BLOCK], block_hash, H256::default()).encode();
-
-	// when
-	let call = crate::create_processed_parentchain_block_call(block_hash, extrinsics);
-
-	// then
-	assert_ne!(call.0, zero_root_call);
-}
```

Comment:
These were moved to the parentchain block importer, and called with `cargo test`.